src/SIPSorceryMedia.FFmpeg.sln

```diff
@@ -21,6 +21,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FFmpegTranscodeTest", "..\t
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FFmpegManagedTranscodeTest", "..\test\FFmpegManagedTranscodeTest\FFmpegManagedTranscodeTest.csproj", "{63DA3C7A-D4FB-4C68-AEE0-91D3EB3CD695}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SIPSorcery", "..\..\sipsorcery\src\SIPSorcery.csproj", "{76FF505E-253C-B75A-7385-7F313F829284}"

```

Comment:
Is this needed? Really need to avoid coupling abck to the main library.

Reply:
no that's the solution file visual studio generated whilst i worked on the 2 projects